app/yolov8-cls-inference/main.cpp

```diff
@@ -0,0 +1,68 @@
+
+#include <fstream>
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/core/mat.hpp>
+#include <opencv2/core/types.hpp>
+#include <opencv2/dnn/dnn.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
+#include <string>
+#include <vector>
+
+int main(int argc, char** argv) {
+  if (argc < 2) {
+    std::cerr << "[fatal error] Path to an input image is missing\n";
+    return 1;
+  }
+
+  // read an input image
+  cv::Mat raw_image = cv::imread(argv[1]);
+  if (raw_image.empty()) {
+    std::cerr << "[fatal error] Failed to read the input image\n";
+    return 1;
+  }
+
+  // read the classification list
+  std::ifstream file("classification_list.txt");
```

Comment:
Does it really work? I mean, your built executable is located in another directory, but you're looking for the .txt file in the directory from which you're running the executable